Jenkinsfile

```diff
@@ -3745,11 +3745,11 @@ assert_frame_equal(training_curve, gt_curve, rtol=1e-3, atol=1e-3)"'''
         model.data.train_ds.concat_sampling_probabilities=[0.3,0.7] \
         model.data.train_ds.num_workers=0 \
         model.data.test_ds.micro_batch_size=1 \
-        model.data.test_ds.global_batch_size=4 \
+        model.data.test_ds.global_batch_size=1 \
```

Comment:
any reasons for changing this? the extra gbs has helped a lot in the past in CI to stop grad acc related bugs from being merged in

Reply:
oh, I'm assuming this is because of the bugs we've had, makes sense